config.ini

```diff
@@ -0,0 +1,6 @@
+[fence]
+CLIENT_ID=REPLACE_ME
+CLIENT_SECRET=REPLACE_ME
+AUTHZ_URI=https://qa.dcf.planx-pla.net/user/oauth2/authorize
+REDIRECT_URI=https://local.broadinstitute.org/fence-callback
+TOKEN_URL=https://qa.dcf.planx-pla.net/user/oauth2/token
```

Comment:
why an ini file instead of environment variables in app.yml?

Reply:
This was done for testing.  Especially in intellij or I think even using nose, we can always read from the .ini, whereas app.yml isn't always loaded.